### DISCUSSION TITLE
How to bind a value to option picker

### DISCUSSION THREAD

**Discussion by Jefflai0315** [Budibase/budibase#4322] [Help]
Is there a way to bind a value/row_id to a option picker?

**Comment by mjashanks** [comment]
You can bind to the `Default Value` of an option picker? This will load the value in when the form loads. Is this what you are asking?

**Reply by Jefflai0315** [reply]
Thanks for the reply.

let me try to explain my situation...
I have created 2 budibase tables and they have a relational field links to one another. 
In the user interface, I want to create two separate forms that the user need to fill. Once the user completed the first form, I want him to fill up the second form while I automatically bind the id of the first form to the second one.

I hope it clarifies.

**Reply by mjashanks** [reply]
Gotcha. So - I was able to do this all inside one form. This is a little annoying, because you will have to give all your fields names custom names. In my video below, by `Contact` table and `Customer` table both have a column  called `Name`.

In my for, I name the fields `Customer_Name` and `Contact_Name`.

Then in my Save button, I do 2 x `Save Row` action - using those fields. The second `Save Row` also uses the `_id` returned from the first, to setup the relationship.

(I also uncovered a bug, whereby the second `Save Row` action is displaying the wrong parameters - but it still works)...

https://user-images.githubusercontent.com/3524181/152634375-caca8a56-f7e4-44fc-98c8-e152163857d8.mp4

**Reply by Jefflai0315** [reply]
Cool trick! But it does not show on my other table. Also, I have created 2 rows in my dababase with this method.( one with customer_name, the other with contact_name)

**Reply by mjashanks** [reply]
Sorry, I thought you wanted to create 2 rows in separate tables. In that case, I'm not sure I understand what you want to do...
